src/App.tsx

```diff
@@ -15,36 +16,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ByLength = 'ByLength',
+  Alphabetically = 'Alphabetically',
+}
+
+function sortGoods(
+  goods: string[],
+  sortType: SortType | null,
+  reverse: boolean,
+): string[] {
+  const sortedGoods = [...goods].sort((good1, good2) => {
+    switch (sortType) {
+      case SortType.Alphabetically:
+        return good1.localeCompare(good2);
+
+      case SortType.ByLength:
+        return good1.length - good2.length;
+
+      default:
+        return 0;
+    }
+  });
+
+  return reverse ? sortedGoods.reverse() : sortedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sort, setSort] = useState<SortType | null>(null);
```

Comment:
```suggestion
  const [sort, setSort] = useState<SortType>(SortType.Default);
```

Create `Default`  option in `enum`